hypothesis-python/src/hypothesis/strategies/_internal/core.py

```diff
@@ -887,7 +887,7 @@ def builds(
         if to_infer - set(hints):
             badargs = ", ".join(sorted(to_infer - set(hints)))
             raise InvalidArgument(
-                f"passed infer for {badargs}, but there is no type annotation"
+                f"passed ... for {badargs}, but there is no type annotation"
```

Comment:
Without the word "infer" this might be a confusing message - how about "but we cannot infer a strategy because there is no type annotation"?